test/test_mail.py

```diff
@@ -557,3 +564,26 @@ def test_disable_tracking(self):
     def test_directly_setting_substitutions(self):
         personalization = Personalization()
         personalization.substitutions = [{'a': 0}]
+
+    def test_from_emailmessage(self):
+        message = EmailMessage()
+        body = 'message that is not urgent'
+        try:
+            message.set_content(body)
+        except AttributeError:
+            # Python2
+            message.set_payload(body)
+        message.set_default_type('text/plain')
+        message['Subject'] = 'URGENT TITLE'
+        message['From'] = 'test@example.com'
+        message['To'] = 'test@sendgrid.com'
+        mail = Mail.from_EmailMessage(message)
+        self.assertEqual(mail.subject, 'URGENT TITLE')
+        self.assertEqual(mail.from_email.email, 'test@example.com')
+        self.assertEqual(len(mail.personalizations), 1)
+        self.assertEqual(len(mail.personalizations[0].tos), 1)
+        self.assertDictEqual(mail.personalizations[0].tos[0], {'email': 'test@sendgrid.com'})
```

Comment:
Python docs show that `assertEqual` will already use `assertDictEqual` to compare dicts by default and that it isn't necessary to call it directly.

* https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertDictEqual

Reply:
Good to know! I'll make the correction.